.vib/mariadb-galera/ginkgo/mariadb_test.go

```diff
@@ -70,17 +74,16 @@ var _ = Describe("MariaDB Galera", Ordered, func() {
 				return c.BatchV1().Jobs(namespace).Get(ctx, createDBJobName, getOpts)
 			}, timeout, PollingInterval).Should(WithTransform(getSucceededJobs, Equal(int32(1))))
 
-			By("scaling down to 0 replicas")
-			ss, err = utils.StsScale(ctx, c, ss, 0)
-			Expect(err).NotTo(HaveOccurred())
+			By("running rollout restart")
+			// Annotate pods to force a rollout restart
+			ss, err = utils.StsAnnotateTemplate(ctx, c, ss, restartAnnotation)
 
-			Eventually(func() (*appsv1.StatefulSet, error) {
-				return c.AppsV1().StatefulSets(namespace).Get(ctx, stsName, getOpts)
-			}, timeout, PollingInterval).Should(WithTransform(getAvailableReplicas, BeZero()))
-
-			By("scaling up to the original replicas")
-			ss, err = utils.StsScale(ctx, c, ss, origReplicas)
-			Expect(err).NotTo(HaveOccurred())
+			// Wait for the new annotation in the existing pods
+			for i := int(origReplicas) - 1; i >= 0; i-- {
```

Comment:
Mariadb-galera pods takes around 1 min to spin up and the rollout restart rebuilds the pods from the last one (mariadb-galera-2 > mariadb-galera-1 > mariadb-galera-0). If we start the loop in the first pod we can face a timeout.